examples/src/RecordMovieComponent.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect, useRef } from "react";
+
+interface RecordMovieComponentProps {
+    isRecording: boolean;
+    setIsRecording: (isRecording: boolean) => void;
+}
+
+const RecordMovieComponent = ({
+    isRecording,
+    setIsRecording,
+}: RecordMovieComponentProps) => {
+    let [recordingDuration, setRecordingDuration] = useState<number>(0);
+    let [recordingStatus, setRecordingStatus] =
+        useState<string>("Not recording");
+    let [outputStatus, setOutputStatus] = useState<string>("");
```

Comment:
Same here, this could also be a bool/enum

Reply:
Except that I want it to be a `Finished recording` message on `stop`, but an empty string on page load which is also `isRecording = false`

Reply:
Okay, keeping it as a string is fine for now! If this needs to capture more states (or if the same message string is being reused) then it's probably worth turning into an enum in the future.